File: unit2/java/project/Risk.java
import java.util.Scanner;

public class Risk {

    public static void main(String[] args) {
        float a;
        float b;
        float c;
        float d;
        String food;
        Scanner scan = new Scanner(System.in);
        System.out.print("What food is being investigated? ");
        food = scan.next();
        System.out.println(food);
        System.out.print("How many people ate " + food + " and got sick? ");
        a = scan.nextInt();
        System.out.println(a);
        System.out.print("How many people ate " + food + " and did NOT get sick? ");
        b = scan.nextInt();
        System.out.println(b);
        System.out.print("How many people did NOT eat " + food + " and got sick? ");
        c = scan.nextInt();
        System.out.println(c);
        System.out.print("How many people did NOT eat " + food + " and did NOT get sick? ");
        d = scan.nextInt();
        System.out.println(d);
        System.out.println("The relative risk of " + food + " is:");
        float riskExposed = a / (a + b);
        float riskUnexposed = c / (c + d);
        float relativeRisk = riskExposed / riskUnexposed;
        System.out.println(relativeRisk);
    }

}

File: unit2/python/project/risk.py
def main():
    food = input("What food is being investigated? ")
    print(food)
    a = float(input("How many people ate " + food + " and got sick? "))
    print(a)
    b = float(input("How many people ate " + food + " and did NOT get sick? "))
    print(b)
    c = float(input("How many people did NOT eat " + food + " and got sick? "))
    print(c)
    d = float(input("How many people did NOT eat " + food + " and did NOT get sick? "))
    print(d)
    print("The relative risk of " + str(food) + " is:")
    risk_exposed = a / (a + b)
    risk_unexposed = c / (c + d)
    relative_risk = risk_exposed / risk_unexposed
    print(relative_risk)


if __name__ == "__main__":
    main()
